projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -617,17 +617,21 @@ class="wpas-submit-<?php echo esc_attr( $connection_data['service_name'] ); ?>"
 			$title = '';
 		}
 
+		$is_social_note = 'jetpack-social-note' === get_post_type( $post->ID );
```

Comment:
Similar to the other PR. I wonder what we can do to not have to use this string in lots of places.

I wonder if we can define support on the post type for something like `publicize-custom-message`, but we can refactor it in a follow up.

Reply:
Yeah that would be a good idea for the future